well_profile/generator.py

```diff
@@ -32,23 +32,19 @@ def two_points(points):
              'north': point_2['north'] - point_1['north'],
              'east': point_2['east'] - point_1['east']}
 
+
+
     delta['horizontal'] = (delta['north']**2 + delta['east']**2)**0.5
 
     # Define azimuth
     azimuth = 0
-    if delta['north'] != 0 and delta['east'] != 0:
-        azimuth = degrees(atan(abs(delta['east']) / abs(delta['north'])))
+    #if delta['north'] != 0 and delta['east'] != 0:
+    beta = degrees(atan(delta['north'] / delta['east']))
```

Comment:
like a lot this change 👍🏻, but it's raising error when `delta['east']=0`

A suggestion:

```
# Define azimuth
if delta['east'] != 0:
      beta = degrees(atan(delta['north'] / delta['east']))
      if delta['east']>=0:
              azimuth = 90-beta
      else:
              azimuth = 270-beta
else:
       if delta['north'] > 0:
              azimuth = 0
       else:
              azimuth = 180
```

Reply:
Divide by zero.

Us [atan2](https://docs.python.org/3/library/math.html#math.atan2).

Reply:
atan2 is a good solution to avoid the issue "divide by zero" but it's not always returning the right value for azimuth, so would need some tweaking anyway